src/Octoshift/Handlers/GrantMigratorRoleCommandBaseHandler.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.Commands;
+using OctoshiftCLI.Contracts;
+
+namespace OctoshiftCLI.Handlers;
+
+public class GrantMigratorRoleCommandBaseHandler
+{
+    private readonly OctoLogger _log;
+    private readonly ITargetGithubApiFactory _githubApiFactory;
+
+    public GrantMigratorRoleCommandBaseHandler(OctoLogger log, ITargetGithubApiFactory githubApiFactory)
+    {
+        _log = log;
+        _githubApiFactory = githubApiFactory;
+    }
+    public async Task Handle(GrantMigratorRoleCommandArgs args)
```

Comment:
nit: missing new line.

Reply:
huh I would have expected dotnet format to catch that. fixing.